routes/video.js

```diff
@@ -1,13 +1,15 @@
 const express = require("express");
 const videoRouter = express.Router();
-const { getVideoList, createVideo } = require("../controllers/video");
+const {
+  getVideoList,
+  createVideo,
+  updateVideo,
+} = require("../controllers/video");
 const { uploadVideo } = require("../routes/middlewares/uploadVideo");
```

Comment:
uploadVideoToAWS 로 수정 부탁드립니다.
```suggestion
const { uploadVideoToAWS } = require("../routes/middlewares/uploadToAWS");
```